### DISCUSSION TITLE
Extend `default:semanticPrefixChore` or `:semanticPrefixChore`?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#29714] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

When you use a config preset from the `default` group, like [`:semanticPrefixChore`](https://docs.renovatebot.com/presets-default/#semanticprefixchore) which is correct?

1. Put `:semanticPrefixChore` in the `extends` array?
2. Put `default:semanticPrefixChore` in the `extends` array?

I always get confused with the `default` group preset, because all the other presets _have_ a prefix, for example: `config:recommended` or `helpers:pinGitHubActionDigests`.

Maybe it would be easier to be consistent throughout, and always use the pattern: `presetGroup:specificPresetName`? That way you always have _one_ way to point to a preset.

What's the reason for having the shorthand, like `:semanticPrefixChore`?

### Logs (if relevant)

Not relevant to my question.

**Comment by rarkins** [comment]
The reason was simply to keep it short

**Reply by HonkingGoose** [reply]
What do you think of my idea to add the `default` bit in front, so it matches the other presets?

**Reply by rarkins** [reply, marked as answer]
I prefer to leave it as is - has been working this way for 6 years now
